extra/1-currency-conversion.js

```diff
@@ -5,7 +5,11 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(amount) {
+  return amount * 1.4;
+
```

Comment:
This is a very small comment but it is customary to not add any extra lines if not needed, so you can erase these lines.

Reply:
Hi @mcarballopacheco Thanks for your comment, could you please clarify which lines I can erase?
Thanks)